.github/workflows/debian.yml

```diff
@@ -0,0 +1,36 @@
+name: Debian packages
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: '0'
```

Comment:
```suggestion
```

https://github.com/actions/checkout#usage
```
# Number of commits to fetch. 0 indicates all history for all branches and tags.
# Default: 1
fetch-depth: ''
```

Reply:
Thanks, this is copy-paste from somewhere. I'm not sure if we depth=1 is enough since we need `git describe --tags` to work properly.

Reply:
Yeah, doesn't seem to work without this